src/Microsoft.Health.Fhir.SqlServer/Microsoft.Health.Fhir.SqlServer.csproj

```diff
@@ -56,8 +56,14 @@
     <EmbeddedResource Include="Features\Schema\Migrations\18.sql">
       <InputToImmutableSqlGenerator>true</InputToImmutableSqlGenerator>
       <InputToMutableSqlGenerator>true</InputToMutableSqlGenerator>
+      <MutableClassVersion>18</MutableClassVersion>
+    </EmbeddedResource>
+    <EmbeddedResource Include="Features\Schema\Migrations\19.sql">
+      <InputToImmutableSqlGenerator>true</InputToImmutableSqlGenerator>
       <MutableClassVersion>Latest</MutableClassVersion>
+      <InputToMutableSqlGenerator>true</InputToMutableSqlGenerator>
     </EmbeddedResource>
+    <EmbeddedResource Include="Features\Schema\Migrations\19.diff.sql" />
```

Comment:
Super nit: can we make the ordering here consistent with the rest of the file?

Reply:
The order has been updated.